systemvm/patches/debian/config/opt/cloud/bin/configure.py

```diff
@@ -145,40 +145,41 @@ def add_rule(self, cidr):
             logging.debug("Current ACL IP direction is ==> %s", self.direction)
             if self.direction == 'egress':
                 self.fw.append(["filter", "", " -A FW_OUTBOUND -j FW_EGRESS_RULES"])
+                fwr = " -I FW_EGRESS_RULES"
+
+                #In case we have a default rule (accept all or drop all), we have to evaluate the action again.
+                if self.rule['cidr'] == ['0.0.0.0/0']:
```

Comment:
Here the default rule when egress policy is true is identified using the cidr '0.0.0.0/0'. What if we configure a egress rule  (after the network is up ) with cidr '0.0.0.0/0' (ex: cidr 0.0.0.0/0, tcp, 22 egress policy on network:true)